joomla/joomla-cms
# Fix invalid string that causes the ini file not to load

Pull Request for Issue #13541 
closes #13541

### Summary of Changes

remove slash

### Testing Instructions

Attempt to install Joomla from staging branch in da-DK language
See that english language is shown
apply this PR
see that on reload the danish language is now loaded

### Documentation Changes Required

None
